### DISCUSSION TITLE
heltec lora32 V3 with rak7289V2

### DISCUSSION THREAD

**Discussion by hoka83** [meshtastic/firmware#3738] [Q&A] [answered]
Hello , is this library compatible with RAK7289V2 and Heltec Lora32 V3? , i want to connect heltec lora32 V3 with RAK7289V2 gateway and send a sketch of "hello world" for example to RAK7289V2 gateway using arduino ide

**Comment by garthvh** [comment, marked as answer]
Lorawan gateways are not meshtastic compatible

**Reply by hoka83** [reply]
so there is no solution to connect between to devices ?

**Reply by garthvh** [reply]
There is not

**Reply by garthvh** [reply]
You would have to bridge them using mqtt
